.github/workflows/reusable-e2e.yml

```
@@ -0,0 +1,39 @@
+          bun-version: ${{ inputs.bun-version }}
+
+      - name: 📦 Install dependencies
+        run: bun install
+
+      - name: 🧪 Run E2E tests
+        id: e2e-test
+        run: bun test e2e.test.ts
+
+      - name: 📤 Set result output
+        id: set-result
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:57: Double quote to prevent globbing and word splitting

(shellcheck)